Sources/AblyChat/DefaultMessages.swift

```diff
@@ -0,0 +1,265 @@
+import Ably
+
+
+public struct MessageEventPayloadWrapper: Hashable, Sendable {
```

Comment:
What does this do (documentation?) and why is it public?

Reply:
removed in favour of `MessageSubscription`